src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1028,9 +1028,7 @@ void checkSingleOverriddenMember(Symbol overridingMember, Symbol overriddenMembe
                                 overridingMemberLocation,
                                 overriddenMethod,
                                 overridingMethod,
```

Comment:
Was the `checkReturnType` value superfluous before?

Reply:
It was only used to suppress nullable variance diagnostics related to the return type. The only time false was passed for it was for a property setter, which is going to return void anyway, so we don't really expect any nullable diagnostics related to the return type to occur.

Reply:
So, I hate to be the voice of insanity, but IL doesn't require that `set` be a `void`-returning method. What do we do if we're overriding a non-`void` setter?

Reply:
This change may cause redundant or strange nullable diagnostics to appear in that case. I am OK with that.